String_Pangram.py

```diff
@@ -1,8 +1,13 @@
-from string import ascii_lowercase as asc_lower
-def check(s):
-    return set(asc_lower) - set(s.lower()) == set([])
-str=raw_input("String:")
```

Comment:
remove "raw_input" don't copy past it from web. Your raw_input was not defined in the program any where how is it working for you.